tests/device_tests/evolu/test_evolu_commands.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from trezorlib import evolu
+from trezorlib.debuglink import SessionDebugWrapper as Session
+
+# make -C core test_emu TESTOPTS="-k test_evolu_commands.py"
+
+pytestmark = pytest.mark.models("safe")
+
+
+@pytest.mark.models(skip="thp")
```

Comment:
use protocol pytest mark instead
```suggestion
@pytest.mark.protocol("protocol_v1")
```